apps/web/src/pages/invites/components/MembersTable.cy.tsx

```diff
@@ -1,7 +1,7 @@
 import { MembersTable } from './MembersTable';
 import { TestWrapper } from '../../../testing';
 import { MemberRoleEnum, MemberStatusEnum } from '@novu/shared';
-import { SinonSpy } from 'cypress/types/sinon';
+import { SinonSpy } from 'sinon';
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
🗒 **note (non-blocking):** This broke when changing module resolution in tsconfig -- also just not ideal anyway‏